### DISCUSSION TITLE
Hello, I'm here to learn with you

### DISCUSSION THREAD

**Discussion by gazimaksudur03** [community/community#124514] [GitHub Education]
### Hi everyone,

how to get proper guidance to use github in my daily purpose?

**Comment by CopperEagle** [comment]
Hey @gazimaksudur03, welcome to Github :tada: 

If you are new to GitHub, have a look at the [Hello World page](https://docs.github.com/en/get-started/start-your-journey/hello-world) :smile: Also, here is the [documentation](https://docs.github.com/en/get-started).

Whenever you have any specific questions, feel free to create new discussions in the "New to Github" category :)
